omc3/plotting/plot_optics_measurements.py

```diff
@@ -416,6 +514,45 @@ def _plot_param(optics_parameter, files, file_labels, x_column, x_label, ip_posi
                            'share_xaxis'])
     )
 
+def _plot_norm_dispersion(optics_parameter, files, file_labels, x_column, x_label, ip_positions, opt):
+    """Plotting normalized dispersion function for normalized dispersion.
+    Normalized dispersion is special, as we only have X. 
+    We therefore plot the delta in the second plot. """
```

Comment:
Bit repetitive with "normalized dispersion". For the second plot, do you mean second axis?

Reply:
```python
 """Plotting normalized dispersion function for normalized dispersion.
    Normalized dispersion is special, as we only have horizontal normalized dispersion. 
    We therefore plot the delta of normalized dispersion in the second plot in normalized dispersion plots. """
    ```

Reply:
With second plot, Josch probably means the bottom plot of our standard two-plot windows in the GUI (and two-plot matplotlib outputs), usually showing X / Y, real / imag, or amp / phase.

+1 for using this unused space for the delta.